mozilla-release/mobile/android/confvars.sh

```diff
@@ -41,4 +41,4 @@ MOZ_USE_NATIVE_POPUP_WINDOWS=1
 MOZ_APP_ID={aa3c5121-dab2-40e2-81ca-7ea25febc110}
 
 # Enable checking that add-ons are signed by the trusted root
-MOZ_ADDON_SIGNING=1
+MOZ_ADDON_SIGNING=`[ -n "${CLIQZ_DEBUG}" ] && echo 0 || echo 1`
```

Comment:
how about `MOZ_ADDON_SIGNING="${CLIQZ_DEBUG=0}"`  ?

Reply:
Well, it's a different semantic, what I want to have is:
CLIQZ_DEBUG=<anyvalue> -> MOZ_ADDON_SIGNING=0
CLIQZ_DEBUG unset or empty -> MOZ_ADDON_SIGNING=1

Reply:
@chrmod probably this is better:
```bash
MOZ_ADDON_SIGNING=`[ "$CLIQZ_DEBUG" -ne 1 ] && echo 1 || echo 0`
```

Reply:
:+1: